x-pack/plugins/rollup/public/crud_app/store/actions/create_job.js

```diff
@@ -65,14 +63,13 @@ export const createJob = (jobConfig) => async (dispatch) => {
     return;
   }
 
-  toastNotifications.addSuccess(`Rollup job '${jobConfig.id}' was created`);
```

Comment:
This change is unrelated to the point behind this PR, but it's a minor UX improvement that I need to get in.